README.md

```diff
@@ -502,32 +502,34 @@ int64_t rdtsc() {
   return static_cast<int64_t>(__rdtsc());
 }
 
-static double compute_cost(double measurement) {
-  // convert to some representative cost
-  return measurement;
+static void BM_NuancedManualTiming(benchmark::State& state) {
+	int64_t rotations = state.range(0);
+	// select a good amount of re-runs to batch elements
+	// using the KeepRunningBatch() API
+	// divide by the # of rotations
+	// since larger rotations
+	// mean smaller necessary batch sizes
+	// for good measurements!
+	size_t batch_size = static_cast<size_t>(10000000 / rotations);
+	std::unique_ptr<int32_t[]> data(new int32_t[rotations]);
+	while (state.KeepRunningBatch(batch_size)) {
+		for (size_t i = 0; i < batch_size; ++i) {
+			auto start = rdtsc();
```

Comment:
Why is it imperative that `rdtsc` is called every iteration, and not just at the start and end of the benchmark?

Reply:
I am building this example based on my understanding of KeepRunningBatch, which is probably very poor right now.

My impression was I needed to do batch_size number of iterations and, when I set UseManualTime (even without this PR), I am required to submit 1 measurement per iteration. If that's not the case I can submit times for batch_size iterations, or maybe just surrounding the whole thing?

I am unsure how KeepRunningBatch works in the presence of UseManualTime, I could probably spend a few more days checking to be sure....